dtcInterfaceLib/CMakeLists.txt

```diff
@@ -1,17 +1,19 @@
 set(NODEJS_ADDON_NAME "DTC")
 
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+include_directories(./)
```

Comment:
I don't think we want to do this, rather includes should be fixed to have dtcInterfaceLib/ in them

Reply:
- makes sense,  it is not even currently needed (might've been needed for generating python interface, but that didn't work out) 
- It looks that it was more learning of cmake... 
- I updated the pull request with the fixed file